docs/explanation/selecting-populations-for-study.md

```diff
@@ -0,0 +1,28 @@
+This page is aimed at researchers working with NHS England patient record data
+as provided by the OpenSAFELY backends.
+
+## Continuity of patient data
+
+Within the NHS in England, it is usually assumed that
+a patient's current primary care record is complete:
+when a patient moves practice, their record moves with them.
+
+However, a given NHS data source may only cover a subset of patients in England:
+for example, only those included in a particular EHR vendor's database.
+Therefore, where a patient has moved to a practice outside the database,
+that patient would usually be censored from that date onwards.
```

Comment:
You're correct that it would be better to be more precise.

It's not my original wording: instead taken from a design document.

My best guess is I _think_ that it's (a), which also probably warrants (b) as mentioned in the following paragraph.

So maybe:

> Therefore, when a patient moves to a practice outside the database,
> that patient's data would usually be missing in that database from that date onwards.

It would be good for someone else to confirm (or improve!) this wording.

Reply:
Have asked in #research.

Reply:
@LindaNab responded, so going to use that wording.